src/github.com/getlantern/enproxy/proxy.go

```diff
@@ -347,14 +353,6 @@ func clientIpFor(req *http.Request) string {
 	return strings.TrimSpace(ips[0])
 }
 
-func badGateway(resp http.ResponseWriter, msg string) {
-	respond(http.StatusBadGateway, resp, fmt.Sprintf("Enproxy server responding Bad Gateway: %s", msg))
-}
-
-func forbidden(resp http.ResponseWriter, msg string) {
-	respond(http.StatusForbidden, resp, fmt.Sprintf("Enproxy server responding Forbidden: %s", msg))
-}
-
```

Comment:
Getting rid of these convenience functions forces the caller to think a bit more about the response code and error, which I think is good.